lib/Factory/ModuleFactory.php

```diff
@@ -264,10 +264,11 @@ public function getEnabled(): array
      * Get module by Type
      * this should return the first module enabled by the type specified.
      * @param string $type
+     * @param array $properties
```

Comment:
Can we call this `$conditions` instead and write a short comment about it what it does?